app/ios/Podfile.lock

```diff
@@ -362,6 +362,8 @@ PODS:
   - React-jsinspector (0.66.5)
   - React-logger (0.66.5):
     - glog
+  - "react-native-attestation (1.0.0-alpha.99+1f2c467)":
```

Comment:
What is this `+hash` at the end of the version?

Reply:
Just something added by yarn so I'll leave it.

Reply:
It is actually a lerna thing:
https://github.com/lerna/lerna/tree/main/libs/commands/publish#--canary